src/features/SearchResults/useSearchResults.js

```diff
@@ -0,0 +1,45 @@
+import axios from "axios";
+import { useState, useEffect } from "react";
+import { url_back, url_front_search } from "../../common/API/requests";
+import { useQueryParameter } from "../../common/Header/SearchBar/queryParameters";
+import { useLocation } from "react-router-dom/cjs/react-router-dom.min";
+
+
+export const useSearchResults = () => {
+    const query = useQueryParameter("query");
+
+    const location = useLocation();
+    const isMoviesPage = location.pathname.startsWith("/movies");
+
+    const [searchResults, setSearchResults] = useState({
+        status: "loading",
+        data: [],
+    });
+
+    useEffect(() => {
+        setSearchResults({ status: "loading", data: [] });
+
+        const debounce = setTimeout(async () => {
+            if (query) {
+                const url = `${url_front_search}${isMoviesPage ? "movie" : "person"}${url_back}&query=${query}&page=1`;
+
+                try {
+                    const response = await axios.get(url);
+                    setSearchResults({
+                        status: "success",
+                        data: response.data
+                    });
+                } catch (error) {
+                    setSearchResults({
+                        status: "error",
```

Comment:
In case of an error, you set the searchResults state to { status: "error" }, but you do not store error information. It might be useful for debugging or displaying an error message to the user to also save the error to the state:
setSearchResults({
    status: "error",
    error: error.message, 
});

Reply:
You're right, I've added error.message to every hook in the app